src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRentalRate = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  const longTermRental = 7;
+  const shortTermRental = 3;
```

Comment:
name these constants in uppercase

Reply:
Yes, that is a good idea, so when we have more code we can see the difference between variables and constants.